src/admin/python/lsst/qserv/admin/qservCli/launch.py

```diff
@@ -32,11 +32,11 @@
 from typing import Dict, List, Optional
 
 from .opt import (
-    dashboard_port_ev,
-    mariadb_image_ev,
-    qserv_image_ev,
-    dh_user_ev,
-    dh_token_ev,
+    envval_dashboard_port,
```

Comment:
Slight preference for the slightly shorter `env_` prefix here throughout, but leave to your discretion :-)

Reply:
Good point! Will fix that.